testsuite/cluster-test/src/suite.rs

```diff
@@ -7,9 +7,9 @@ use std::{cmp::min, env};
 use crate::{
     cluster::Cluster,
     experiments::{
-        CpuFlamegraphParams, Experiment, ExperimentParam, PerformanceBenchmarkParams,
-        PerformanceBenchmarkThreeRegionSimulationParams, RebootRandomValidatorsParams,
-        RecoveryTimeParams, TwinValidatorsParams,
+        ComaptiblityTestParams, CpuFlamegraphParams, Experiment, ExperimentParam,
```

Comment:
Looks like there is type in `ComaptiblityTestParams` :(

Reply:
LOL nice catch, I'll fix that 😅